src/envo/envo-edit.owl

```diff
@@ -28119,6 +28142,155 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> <http://purl.obolibrar
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000057> <http://purl.obolibrary.org/obo/ENVO_00000298>))
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000032> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002234> <http://purl.obolibrary.org/obo/ENVO_01000293>))
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000033> (marine sediment)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Pelagic_sediment#_note-8") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000033> "Sediment which accumulates within an ocean basin.")
```

Comment:
A material entity can't have a participant, that's for processes.

Also, rephrase:
```
Definition: Sediment which has been transported through the marine water column, settling on the seafloor.

Comment: Particles of marine sediment may have been primarily generated by 1) processes in terrestrial systems and transported to the marine realm by the action of rivers or aeolian processes (amongst other routes) , 2) marine organisms, 3) chemical processes in seawater, or 4) cosmogeneous input.
```